project_task_code/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sequential Code for Tasks",
+    "version": "12.0.1.1.0",
+    "category": "Project Management",
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "http://www.avanzosc.es",
```

Comment:
@dbo-odoo IMO Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)